projects/opendr_ws_2/src/opendr_perception/opendr_perception/skeleton_based_action_recognition_node.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import torch
+import numpy as np
+
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+from vision_msgs.msg import ObjectHypothesis
+from sensor_msgs.msg import Image as ROS_Image
+from opendr_ros2_bridge import ROS2Bridge
+from opendr_ros2_messages.msg import OpenDRPose2D
+
+from opendr.engine.data import Image
+from opendr.perception.pose_estimation import draw
+from opendr.perception.pose_estimation import LightweightOpenPoseLearner
+from opendr.perception.skeleton_based_action_recognition import SpatioTemporalGCNLearner
+from opendr.perception.skeleton_based_action_recognition import ProgressiveSpatioTemporalGCNLearner
+
+
+class SkeletonActionRecognitionNode(Node):
+
+    def __init__(self, input_rgb_image_topic="/usb_cam/image_raw",
```

Comment:
Default usb camera topic is different in ROS2.

```suggestion
    def __init__(self, input_rgb_image_topic="image_raw",
```